<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="LAPTOP-5G0H4FFG" name="TestRunner" tests="1" failures="1" timestamp="2021-10-28T14:04:58 PDT" time="6.149" errors="0">
  <testcase name="runScenario" time="6.149" classname="TestRunner">
    <failure type="java.lang.AssertionError" message="expected [Thank you for sign up] but found [Thank you for sign up, here your id - TTCQj57u]">
      <![CDATA[java.lang.AssertionError: expected [Thank you for sign up] but found [Thank you for sign up, here your id - TTCQj57u]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at pageObject.signup_page.getThankYouMsg(signup_page.java:103)
at stepDef.signup_steps.iShouldSeeThankYouForSignUpMessage(signup_steps.java:41)
at âœ½.I should see Thank You for sign up message(file:///C:/Users/lmakn/Documents/TalentTEK/Git/Sep2021_cucumberBDD/src/test/java/features/signup.feature:8)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- TestRunner -->
